lib/transmuxer/muxjs_transmuxer.js

```diff
@@ -207,6 +211,8 @@ shaka.transmuxer.MuxjsTransmuxer = class {
    * @export
    */
   transmux(data) {
+    goog.asserts.assert(!this.muxTransmuxer_,
```

Comment:
You should assert `this.muxTransmuxer_`, not `!this.muxTransmuxer_`.

Reply:
Ups...I was too fast...